### PR TITLE
Onboarding Stepper Component

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

Adds stepper component to the onboarding sidebar.

[Figma](https://www.figma.com/file/PvL23fm6W36Zd88O1XVyIo/Beacon-DX-UI?type=design&node-id=8897-4796&mode=design&t=skAGJk2i7JCyKxA1-4)

Fixes SC-20488

### Type of change

- [x] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

Ensign by Rotational Labs (41)

### Definition of Done

- [ ] I have manually tested the change running it locally (having rebuilt all containers) or via unit tests 
- [ ] I have added unit and/or integration tests that cover my changes
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have updated the dependencies list if necessary (including updating yarn.lock and/or go.sum)
- [ ] I have recompiled and included new protocol buffers to reflect changes I made if necessary
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [ ] I have notified the reviewer via Shortcut or Slack that this is ready for review
- [x] Front-end: Checked sm, md, lg screen resolutions for effective responsiveness
- [ ] Backend-end: Documented service configuration changes or created related devops stories

### Reviewer(s) checklist

- [ ] Front-end: I've reviewed the Figma design and confirmed that changes match the spec.
- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: web/beacon-app/src/features/auth/components/Register/SuccessfullAccountCreation.tsx [hunk 1]
```diff
@@ -40,7 +40,11 @@ function SuccessfullAccountCreation() {
               .
             </Trans>
           </p>
-          <Button variant="primary" className="mt-4 font-bold text-white">
+          <Button
+            variant="primary"
+            onClick={() => console.log('Resend verification message!')}
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/auth/components/Register/SuccessfullAccountCreation.tsx:45]: Added this following a comment from a previous PR review.

File: web/beacon-app/src/features/onboarding/components/OnboardingStepperStep.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Trans } from '@lingui/macro';
+
+interface OnboardingStepperStepProps {
+  title: string;
+  description: string;
+}
+
+const OnboardingStepperStep = ({ title, description }: OnboardingStepperStepProps) => {
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/onboarding/components/OnboardingStepperStep.tsx:8]: I didn't see a Stepper component that could be imported from `flowbite` or `flowbite-react`. The code is from Flowbite's site with some minor changes to make the step indicator smaller.

File: web/beacon-app/src/features/onboarding/components/OnboardingStepperStep.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <>
+      <li className="mb-10 ml-6">
+        <span className="absolute -left-[4px] mt-1 flex h-2 w-2 items-center justify-center rounded-full bg-gray-100 ring-4 ring-white"></span>
+        <h3 className="font-medium leading-tight">
+          <Trans>{title}</Trans>
+        </h3>
+        <button onClick={() => console.log('Step')} className="text-sm">
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/onboarding/components/OnboardingStepperStep.tsx:16]: Using the button component would've required more customization, so using the button element felt like the better approach for now.

**Review by masskoder — APPROVED** [review]
Looks good. I just added a minor update to make the stepper maintainable.

**Pull request merged**